### PR TITLE
Close for v1.1

### PR DESCRIPTION
**Pull request by BryanL2303**
Will continue to update UG once functions are added in v1.2

### CODE REVIEW

**Review by azumieflare — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -189,10 +206,11 @@ _Details coming soon ..._
...
 Action | Format, Examples
 --------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
+**Add employee** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/Ben Diddle t/friend e/bendiddle@example.com a/Newgate Prison p/21092109 t/criminal`
+**List employees** | `listemployees`
+**Delete employee** | `delete EMPLOYEE_ID` <br> e.g., `delete 2`
+**Add task** | `task TASK_DESCRIPTION` <br> e.g., `task Weekly meeting`
+**List tasks** | `listtasks`
+**Delete task** | `deletetask TASK_ID`<br> e.g., `deletetask 3`
 **Clear** | `clear`
```
Comment (gavin331) [inline docs/UserGuide.md:215]: Do we add the command summary for the mark and unmark?

**Review by gavin331 — COMMENTED** [review]
Do we need to add mark and unmark into the command summary?

**Review by gavin331 — APPROVED** [review]
LGTM

**Review by johnyoozhengxian — APPROVED** [review]
All good

**Pull request merged**